README.md

```diff
@@ -434,6 +439,19 @@ is set to `'true'`, it will be used as header value as is, without any extra mod
 | gid.connector.http.sink.writer.request.mode             | optional | Sets Http Sink request submission mode. Two modes are available to select, `single` and `batch` which is the default mode if option is not specified.                                                                                            |
 | gid.connector.http.sink.request.batch.size              | optional | Applicable only for `gid.connector.http.sink.writer.request.mode = batch`. Sets number of individual events/requests that will be submitted as one HTTP request by HTTP sink. The default value is 500 which is same as HTTP Sink `maxBatchSize` |
 
+
+## Lookup Cache 
+The HTTP Client connector can be used in temporal join as a lookup source (aka. dimension table). Currently, only sync lookup mode is supported.
```

Comment:
What do you mean by " only sync lookup mode is supported"?

Reply:
> What do you mean by " only sync lookup mode is supported"?

I copied there words from [https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/jdbc/#lookup-cache](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/jdbc/#lookup-cache) as it is the same caching mechanism. I will expand on you points to make it more clear.